frontend/src/pages/instance/components/InstanceForm.vue

```diff
@@ -11,7 +11,7 @@
         <!-- Form title -->
         <div class="mb-8 text-sm text-gray-500">
             <template v-if="creatingNew">
-                <template v-if="applicationSelection">
+                <template v-if="!creatingApplication || applicationSelection">
```

Comment:
Is this change intended?

Reply:
Yes - albeit slightly off-topic and it slipped in.

I noticed when we're on the 'Create Instance' page (so no Application creation/selection to be done), we were showing the 'Applications are used to ...' explainer. This fixes that to show the header related to creating an instance.